packages/eslint-config-airbnb/test/test-base.js

```diff
@@ -1,13 +1,13 @@
 import test from 'tape';
 import base from '../base';
 
-test('base: does not reference react', t => {
-  t.plan(2);
+test('base: does not reference react', tst => {
```

Comment:
I think they always make sense as iteration variables in loops, and in tape tests, to begin with.